dev-env/update-openapi

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+set -eux
```

Comment:
out of curiosity, any reason not to set the options at the start of the script?

Reply:
Because I didn't want the whole `SCRIPT_DIR` evaluation to be printed (with `-x`). I'll instead do the same as in other scripts.